e2e-tests/pitr/conf/restore-on-pitr-minio.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBClusterRestore
+metadata:
+  name: on-pitr-minio
+spec:
+  pxcCluster: pitr
+  backupName: on-pitr-minio
+  backupSource:
+      storageName: "minio"
+      destination: <destination>
+      s3:
+        bucket: operator-testing
+        credentialsSecret: minio-secret
+        endpointUrl: http://minio-service:9000
+        region: us-east-1
+  pitr:
+    type: latest
+    backupSource:
+      storageName: "minio-binlogs"
```

please don't specify storage and specify only s3 details
```suggestion
  backupSource:
    destination: <destination>
    s3:
      bucket: operator-testing
      credentialsSecret: minio-secret
      endpointUrl: http://minio-service:9000
      region: us-east-1
  pitr:
    type: latest
    backupSource:
      s3:
        bucket: operator-testing/binlogs
        credentialsSecret: minio-secret
        endpointUrl: http://minio-service:9000
        region: us-east-1
```